sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1574,14 +1605,15 @@ def join(self):
 
 
 class DomInfoUpdateTask(threading.Thread):
-    def __init__(self, namespaces, port_mapping, main_thread_stop_event):
+    def __init__(self, namespaces, port_mapping, main_thread_stop_event, skip_cmis_mgr):
         threading.Thread.__init__(self)
         self.name = "DomInfoUpdateTask"
         self.exc = None
         self.task_stopping_event = threading.Event()
         self.main_thread_stop_event = main_thread_stop_event
         self.port_mapping = copy.deepcopy(port_mapping)
         self.namespaces = namespaces
+        self.skip_cmis_mgr = skip_cmis_mgr
```

Comment:
@mihirpat1 I am not sure if this is a must required because platform which skip cmis mgr will return CMIS_STATE_UNKNOWN which means `is_port_in_cmis_initialization_process()` will return False always (even though the fw could be initializing the CMIS module like in SN 2700)

Reply:
@prgeor The function `is_port_in_cmis_initialization_process` currently assumes that `CMIS_STATE_UNKNOWN` is a transient state and hence, returns True.
For platforms which have CMIS manager enabled, we assume that `CMIS_STATE_UNKNOWN` is the first state after starting CMIS state machine post xcvrd boot-up. Hence, the DomInfoUpdateTask thread holds off polling on the corresponding port to allow the port go through CMIS initialization if needed.